controller/src/main/java/io/pravega/controller/fault/ControllerClusterListener.java

```diff
@@ -47,16 +47,17 @@
     private final List<FailoverSweeper> sweepers;
 
     public ControllerClusterListener(final Host host, final Cluster cluster,
-                                     final ScheduledExecutorService executor, final FailoverSweeper... sweepers) {
+                                     final ScheduledExecutorService executor, final List<FailoverSweeper> sweepers) {
```

Comment:
This list could be mutable, which would cause issues if the caller changed it after the fact. Either restrict this type to ImmutableList or make a defensive copy.